### DISCUSSION TITLE
How do you Logout?

### DISCUSSION THREAD

**Discussion by hawaiidude** [prasathmani/tinyfilemanager#587] [Q&A]
How do you Logout?

**Comment by prasathmani** [comment]
logout option will display when click on username
![image](https://user-images.githubusercontent.com/11345598/123717567-bc42f500-d89a-11eb-8547-a62e8504ca7b.png)

**Comment by thrax-github** [comment]
How can I logout a sleeping user ?

**Reply by prasathmani** [reply]
Tinyfilemanager does not have option to logout sleeping users. but you can do this using PHP session timeout configuration

**Reply by prasathmani** [reply]
https://stackoverflow.com/questions/9904105/php-sessions-default-timeout
